run_go_test.sh

```diff
@@ -15,10 +15,14 @@ trap saveResults EXIT
 
 mkdir "${results_dir}" || true
 
+if [[ -z "${TEST_FOCUS}" ]]; then
+  TEST_FOCUS=".*"
```

Comment:
if it's empty, this will assign the default value as `.*` (all tests)

Reply:
Not sure about TEST_FOCUS as the name. What about TEST_PATTERN?

It kind of aligns with the docs.

```
go help test

...
'Go test' recompiles each package along with any files with names matching
the file pattern "*_test.go".
...
```

Also lets make sure we get a review from Celestial too.

Reply:
I tried to align with the sonobuoy docs. 
```
The most common point of customization is changing the set of tests to run. This is controlled by two environment variables the test image recognizes:

E2E_FOCUS
E2E_SKIP
```
https://sonobuoy.io/docs/v0.50.0/e2eplugin/

I don't have any reason to stick with `E2E_FOCUS` so which one you prefer?

Reply:
OK cool I'm fine with aligning with Sonobuoy. I wasn't used to seeing focus used in this context but it makes sense.

Reply:
I'd use the same as the `cncf` tests `E2E_FOCUS`.